scanner/src/funTest/kotlin/storages/PostgresStorageTest.kt

```diff
@@ -38,7 +38,7 @@ class PostgresStorageTest : AbstractStorageTest() {
 
     override fun isolationMode() = IsolationMode.InstancePerTest
 
-    override fun createStorage() = PostgresStorage(postgres.postgresDatabase.connection).also { it.init() }
+    override fun createStorage() = PostgresStorage(postgres.postgresDatabase.connection, "public").also { it.init() }
```

Comment:
Should this be moved to the previous commit?

Reply:
I think it's ok if the history also reflects the order of implementation. In this case schema was hardcoded before it was required to be configurable.